agent/engine/dependencygraph/graph.go

```diff
@@ -115,6 +117,11 @@ func DependenciesAreResolved(target *api.Container,
 		neededVolumeContainers[i] = volume.SourceContainer
 	}
 
+	resourcesMap := make(map[string]taskresource.TaskResource)
+	for _, resource := range resources {
```

Comment:
question: L117 above. the volume dependencies will get tracked through this resources dependency code path eventually yea?

Reply:
Those are dependencies on volume containers and can eventually be modelled as container dependencies.

Reply:
The new volume driver based dependencies will follow the resource dependencies code path.